Ginger/GingerCore/ALM/RQM/RQMConnect.cs

```diff
@@ -401,11 +424,16 @@ public RQMTestPlan GetRQMTestPlanFullData(string RQMServerUrl, string RQMUserNam
                             //
                             // building a list of TestCases
                             LoginDTO loginData = new LoginDTO() { User = RQMUserName, Password = RQMPassword, Server = RQMServerUrl };
-                            IProjectData rqmProjectsData = RQMRep.GetVisibleProjects(loginData);
-                            rqmProjectsDataList = rqmProjectsData.IProjectDefinitions;
+                            if(rqmProjectsDataList == null || rqmProjectsDataList.Count == 0)
```

Comment:
Please add else part to handle what needs to be done of rqmProjectsDataList  is null or empty

Reply:
if rqmProjectsDataList is null then we are fetching data in this and assigne to rqmProjectsDataList other wise data is already present in it then it worked no need to add else part